js/app.js

```diff
@@ -131,7 +59,7 @@ function UserSort(userIdData) {
     }
     return 0;
   });
-  userIdData.push(["tag this person directly in Discord", "(Unlisted User)"]);
+  userIdData = userIdData.unshift(["TagMe", "(New Competitor)"]);
```

Comment:
Note that to prepend (instead of append) an item to an array in JavaScript, you can use this `unshift` method but it returns a new array (instead of adjusting the existing array), thus the change here to need the assign statement